app/models/manageiq/providers/cloud_manager/vm.rb

```diff
@@ -107,7 +107,7 @@ def cpu_percent_available?
     true
   end
 
-  def resize_queue(userid, new_flavor)
+  def resize_queue(userid, new_flavor_id)
```

Comment:
The new request_task is being queued to run with the proper `role` and `zone` so this method is no longer needed by the reconfigure request/task logic.  The task should call the `resize` method directly.

I am not sure if there are any other callers of this method, if not it can be removed.

Reply:
This can be removed. I had already wondered whether I could remove this, but I was holding off until I got feedback on whether that was the right thing to do. I'll pull this method out along with the class renaming for the next iteration.